src/index.js

```diff
@@ -0,0 +1,42 @@
+import express from 'express';
+import trimmer from 'trim-request-body';
+import cors from 'cors';
+import debug from 'debug';
+
+// Instance of express app
+const app = express();
+
+const infoLog = debug('http:info');
+
+// Allow cross origin access
+app.use(cors());
+
+// Parse application/json
+app.use(express.json());
+
+// Parse application/xwww-
+app.use(express.urlencoded({ extended: false }));
+
+// Trim the parsed request body
+app.use(trimmer);
+
+// Handle base route
+app.get('/', (req, res) => {
+    res.status(200).json({
+        status: 200,
+        message: 'Welcome to Barefoot Nomad',
+    });
+});
+
+// Handle routes not found
+app.use('*', (req, res) => {
+    res.status(404).json({
```

Comment:
Can we create a response function in the utils folder. this will allow us to have a defined response format that is reusable in the utils folder or what do you think?

Reply:
I think that makes sense. Please check now, I have made the changes